### ISSUE TITLE
Doesn't open folder in iTerm

### ISSUE THREAD

**Issue by garyking** [wbond/sublime_terminal#157]
Starting a few days ago, when I use "Terminal: Open in project folder", iTerm opens up but it isn't focused, and it doesn't run any commands to `cd` to the project folder. 

I'm using the very latest version of this package and iTerm, and my settings are:

```
{
  "terminal": "iTerm2-v3.sh",
  "parameters": ["--open-in-tab"]
}
```

**Comment by twolfson** [comment]
This is likely related to #150 but I'm not sure what would cause those issues. Are there any messages in the Sublime Text console?

**Comment by garyking** [comment]
No messages appear in the console at all when running this command.

**Comment by twolfson** [comment]
Hmmm, that sure is weird. I will look into reproducing the issue within the next week. If I can't reproduce the error, then it's likely we'll revert the PR as it's been causing a bunch of issues on Windows as well =/

**Comment by twolfson** [comment]
I'm unable to reproduce the error on Linux Mint sadly but I hesitant to ditch the `env` setting as it's quite practical. Instead, I would like to dig a little further into the issue.

@garyking What happens when you run the `iTerm2-v3.sh` script locally?

```bash
# Download Sublime Terminal's iTerm2-v3.sh
wget https://raw.githubusercontent.com/wbond/sublime_terminal/1.16.2/iTerm2-v3.sh

# Enable script as an executable
chmod +x iTerm2-v3.sh

# Run script (maybe run from Terminal so you get fresh start behavior)
./iTerm2-v3.sh --open-in-tab
```

**Comment by garyking** [comment]
After the last command I get:

```
56:136: execution error: iTerm got an error: Can’t get current window. (-1728)
```

**Comment by twolfson** [comment]
What happens when you run the script without `--open-in-tab`?

**Comment by garyking** [comment]
Exact same error. iTerm also still opens.

**Comment by twolfson** [comment]
Weird, this sounds like a problem in the script itself. Can you try debugging it to figure out what's wrong?

**Comment by garyking** [comment]
So far I've noticed that if iTerm is running, then "Open project in Terminal" opens a new tab in iTerm in the project folder just fine. I'll do some more testing later.

**Comment by garyking** [comment]
Problem fixed as of the latest iTerm update 3.0.15 released a few days ago.
